libafl_frida/src/drcov_rt.rs

```diff
@@ -58,8 +60,22 @@ impl DrCovRuntime {
         Self {
             drcov_basic_blocks: vec![],
             ranges: RangeMap::new(),
+            stalked_addresses: HashMap::new(),
         }
     }
+
+    /// Add a stalked address to real address mapping.
+    #[inline]
+    pub fn add_stalked_address(&mut self, stalked: usize, real: usize) {
+        self.stalked_addresses.insert(stalked, real);
+    }
+
+    #[must_use]
+    pub fn real_address_for_stalked(&self, stalked: usize) -> usize {
```

Comment:
Shouldn't this return `Err` or `None` if no real `addr` is available?

Reply:
yeah still wip
I've not tried compiling yet.

Reply:
hmm, I'm not sure.
I'd like to ask s1341